uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java

```diff
@@ -1826,6 +1832,29 @@ public void setAttribute( final Path path,
         view.setAttribute( s[ 1 ], value );
     }
 
+    private void squash( final Path path,
+                         final SquashOption value ) {
+
+        final JGitPathImpl gSource = toPathImpl( path );
+        String branch = getFullBranch( gSource );
+        Repository gitRepo = gSource.getFileSystem().gitRepo().getRepository();
+        String commitMessage = value.getMessage();
```

Comment:
you could checkNotNull/Empty directly here:
String commitMessage = checkNotEmpty("value.message", value.getMessage());